layers/meta-balena-topic/recipes-support/bmap-tools/bmap-tools_3.6.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI = "git://github.com/intel/${BPN};branch=main;protocol=https"
```

Comment:
Do you use bmap-tools? What is the purpose?

Reply:
Not explicitly. We just use the balenaos-img output.
The build does, however, generate some ..rootfs.bmap outputs by default in image_types_balena.bbclass, which could be used with the bmaptool for flashing.

This commit was more for cleaning up the build log to remove warnings.